src/libcore/sync/atomic.rs

```diff
@@ -1550,12 +1550,30 @@ unsafe fn atomic_xor<T>(dst: *mut T, val: T, order: Ordering) -> T {
 
 /// An atomic fence.
 ///
+/// A fence creates happens-before edges between other atomic operations or
```

Comment:
s/edges/dependence/ or s/edges/relantionships/

A fence creates happens-before relationships between not only atomic operations and fences, but also between normal memory operations.